dsb-integration/src/test/java/com/spectralogic/dsbrowser/integration/tasks/Ds3GetJob_Test.java

```diff
@@ -162,7 +162,7 @@ public void createFileMap() throws InterruptedException {
                     "2/07/2017 10:28:17",
                     "spectra",
                     false);
-            final ImmutableList listItems = ImmutableList.builder().add(ds3TreeTableValueCustom).build();
+            final ImmutableList listItems = ImmutableList.of(ds3TreeTableValueCustom);
```

Comment:
This is more intentional. I like the pattern.

Reply:
Ok, good catch.